### PR TITLE
Adding info on write rate limiting

### PR DESCRIPTION
**Pull request by Streadz**
Updating the docs to be explicit in stating that write rate limiting isn't supported.

### CODE REVIEW

**Comment by Streadz** [comment]
👀 @betamoo @garrettheel

File: docs/rate_limited_operations.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@ Rate-Limited Operation
 ================
 
 `Scan`, `Query` and `Count` operations can be rate-limited based on the consumed capacities returned from DynamoDB.
-Simply specify the `rate_limit` argument when calling these methods.
+Simply specify the `rate_limit` argument when calling these methods. Rate limited writes are not currently supported, although workarounds are discussed below.
```
Comment (betamoo) [inline docs/rate_limited_operations.rst]: nit: `rate limited batch writes`

File: docs/rate_limited_operations.rst [hunk 2]
```diff
@@ -62,4 +62,22 @@ You can use `rate-limit` when counting items in your table:
...
+    
+Unsupported Scenarios: Rate limiting Writes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Rate limiting writes are currently not supported. One workaround for this is to keep a count that gets increased based on the time passed, and only batch that number of writes (called Leaky bucket algorithm). The pseudocode below assumes writes
+are happening constantly. In the case that they're not, sleeping during low utilization could also be useful.
+
+.. code-block:: python
+
+    def write(items):
+    while True:
```
Comment (betamoo) [inline docs/rate_limited_operations.rst]: I wrote the code in slack so I didn't get the spacing right 😬 
Can we adjust the spaces

File: docs/rate_limited_operations.rst [hunk 3]
```diff
@@ -62,4 +62,22 @@ You can use `rate-limit` when counting items in your table:
...
+       allowance= min(
+                   allowance + time_passed/RATE_OF_WRITING, 
+                   MAX_ALLOWED_RATE
+                   )
+       number_of_items_to_be_written = min(int(allowance), len(items))
+       batch_write(items[:number_of_items_to_be_written])
+       allowance -= number_of_items_to_be_written
+       items = items[number_of_items_to_be_written:]
```
Comment (betamoo) [inline docs/rate_limited_operations.rst]: let's add
```
#optional
sleep(some_time)
```

File: docs/rate_limited_operations.rst [hunk 4]
```diff
@@ -62,4 +62,22 @@ You can use `rate-limit` when counting items in your table:
...
     print("Count : {}".format(count))
+    
+Unsupported Scenarios: Rate limiting Writes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Rate limiting writes are currently not supported. One workaround for this is to keep a count that gets increased based on the time passed, and only batch that number of writes (called Leaky bucket algorithm). The pseudocode below assumes writes
+are happening constantly. In the case that they're not, sleeping during low utilization could also be useful.
+
+.. code-block:: python
+
+    def write(items):
```
Comment (betamoo) [inline docs/rate_limited_operations.rst]: Let's highlight that this is just a pseudo code

**Comment by Streadz** [comment]
Thanks @betamoo, addressed your feedback!

**Comment by betamoo** [comment]
+1

File: docs/rate_limited_operations.rst [hunk 5]
```diff
@@ -62,4 +62,23 @@ You can use `rate-limit` when counting items in your table:
...
+.. code-block:: python
+    # Note: This is pseudocode
+    def write(items):
+        while True:
+            allowance= min(
+                       allowance + time_passed/RATE_OF_WRITING, 
+                       MAX_ALLOWED_RATE
+                       )
+             number_of_items_to_be_written = min(int(allowance), len(items))
+             batch_write(items[:number_of_items_to_be_written])
```
Comment (garrettheel) [inline docs/rate_limited_operations.rst]: This doesn't take into account the size of the items being written, so I'm a little worried that someone might try to use this algorithm unsuccessfully. I wonder if it's better to just state that this is currently unsupported and add an feature request for it in issues.

Comment (Streadz) [reply docs/rate_limited_operations.rst]: That's fair too. I suppose the workaround advice can be given offline and can be something not actually encouraged so as not to generate additional load if people try it in various configurations and get unexpected results. Updating!

**Comment by Streadz** [comment]
Updated section @garrettheel!

**Comment by garrettheel** [comment]
Thanks @Streadz!

**Pull request merged**